subsys/bluetooth/host/Kconfig

```diff
@@ -349,6 +349,46 @@ config BT_MAX_PAIRED
 	  Maximum number of paired Bluetooth devices. The minimum (and
 	  default) number is 1.
 
+config BT_GAP_PERIPHERAL_PREF_PARAMS
+	bool "Configure peripheral preferred connection parameters"
+	default y
+	depends on BT_PERIPHERAL
+	help
+	  This allows to configure periphral preferred connection parameters.
+	  Enabling this option results in adding PPCP characteristic in GAP
+	  and sending request for connection parameters udpate after GAP
+	  recommended 5 seconds of connection as peripheral. If disabled it is
+	  up to application to set expected connection parameters.
+
```

Comment:
Strictly speaking no since all the entries have a `depends on` for it, however it'd be cleaner to have the `if` whenever there's two or more options depending on it.